### PR TITLE
Small changes to regexp querying

### PR DESCRIPTION
**Pull request by mmotti**
Signed-off-by: MMotti <matthew.w.motti@gmail.com>

**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
Just a quick tidy-up of some code that I have previously submitted

**How does this PR accomplish the above?:**
Makes more sense to use process substitution with mapfile (I think). It makes it possible to check the count of an array properly (`${#regexlist[@]}`) - Here strings add a new line whether the command returned a result or not, so you don't know whether you have one entry or it's just a new line.

Some other small changes to variable names. Using SED instead of AWK to indent output.

Also - I think the 'case' statement at the bottom was a bit of a waste of time; could easily fit within the IF statement that sat just above it.

**What documentation changes (if any) are needed to support this PR?:**
N/A

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

File: advanced/Scripts/query.sh [hunk 1]
```diff
@@ -133,31 +133,37 @@ scanDatabaseTable "${domainQuery}" "whitelist" "${exact}"
...
+			# Echo the "matched" message, indented by one space
+			echo " ${str_message}"
+			# Echo the "results" message, each line indented by three spaces
+			# shellcheck disable=SC2001
+			echo "${str_result}" | sed 's/^/   /'
+		else
+			echo "π Regex list"
+			exit 0
+		fi
+
```
Comment (Mcat12) [inline advanced/Scripts/query.sh]: Extra newline

**Comment by Mcat12** [comment]
Not sure why the CI is failing, might be flaky...

**Comment by mmotti** [comment]
Added an extra comment and removed the whitespace after the if statement.

Not sure why it's failing, either! Think it was failing to download FTL or something when I looked through the details yesterday.

**Comment by Mcat12** [comment]
The CI failure is due to a change from the Fedora 29 to 30 docker image, and is unrelated to this PR.

**Pull request merged**